elements/aura.lua

```diff
@@ -129,20 +127,20 @@ local createAuraIcon = function(icons, index)
 	return button
 end
 
-local customFilter = function(icons, unit, icon, name)
+local function customFilter(icons, unit, icon, name)
 	if((icons.onlyShowPlayer and icon.isPlayer) or (not icons.onlyShowPlayer and name)) then
 		return true
 	end
 end
 
-local updateIcon = function(unit, icons, index, offset, filter, isDebuff, visible)
+local function updateIcon(unit, icons, index, offset, filter, isDebuff, visible)
 	local name, rank, texture, count, dispelType, duration, expiration, caster, isStealable,
 		nameplateShowSelf, spellID, canApply, isBossDebuff, casterIsPlayer, nameplateShowAll,
 		timeMod, effect1, effect2, effect3 = UnitAura(unit, index, filter)
 
 	if(name) then
-		local n = visible + offset + 1
-		local icon = icons[n]
+		local i = visible + offset + 1
```

Comment:
I "fixed" it silently to i on the docs branch, so that I can test later without having to rebase.